py_experimenter/experimenter.py

```diff
@@ -203,14 +328,35 @@ def execution_wrapper(self, approach, custom_fields: dict, keyfields, result_pro
             else:
                 result_processor._change_status('done')
 
-    def reset_experiments(self, status= 'error') -> None:        
-        keyfields,entries = self._dbconnector.delete_experiments_with_status(status)
+    def reset_experiments(self, status= 'error') -> None:
+        """
+        Deletes the experiments of the database table having the given status and adds the according keyfields as new rows.
+        
+        * `created` when the experiment is added to the database table, execution has not started.
+        * `running` when the execution of the experiment has been started.
+        * `error` if something went wrong during the execution, i.e., an exception is raised
+        * `done` if the execution finished successfully.
+
+        :param status: The status of experiments that should be reset. Defaults to 'error'.
```

Comment:
Not sure, if this parameter should have a default. For safety reasons, it would be good if the user would need to specify the status under all circumstances.